common/modules/gcp-project/main.tf

```diff
@@ -201,6 +217,7 @@ data "google_iam_policy" "combined" {
 
     members = [
       "${local.service_accounts}",
+      "serviceAccount:projectowner@${var.project_id}.iam.gserviceaccount.com",
```

Comment:
same  - `projectowner@` is duplicate - once coming from `local.service_accounts` and once explicitly listed above

Reply:
Yup, thanks.